docs/processors/standalone/building.mdx

```diff
@@ -289,6 +289,80 @@ func main() {
 
 Check [Compiling the processor](#compiling-the-processor) for what to do next, and how to compile the processor.
 
+## Schemas
+
+Processors have access to the Schemas in the used Schema Registry using two utility functions:
```

Comment:
Q: is there a special reason for the capital S in `Schemas`?

Reply:
not really no :/ I can change them to "schema", but should I keep "Schema Registry" capitalized?

Reply:
I _think_ `Schema Registry` can stay capitalized (a name).